src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.cpp

```diff
@@ -158,24 +200,88 @@ void CARMACloudPlugin::CARMAResponseHandler(QHttpEngine::Socket *socket)
 	tsm5message.set_contents(container.get_storage().get_tree());
 	tsm5ENC.encode_j2735_message(tsm5message);
 
-	msg.reset();
-	msg.reset(dynamic_cast<tsm5EncodedMessage*>(factory.NewMessage(api::MSGSUBTYPE_TESTMESSAGE05_STRING)));
-
-	string enc = tsm5ENC.get_encoding();
-	msg->refresh_timestamp();
-	msg->set_payload(tsm5ENC.get_payload_str());
-	msg->set_encoding(enc);
-	msg->set_flags(IvpMsgFlags_RouteDSRC);
-	msg->addDsrcMetadata(172, 0x8003);
-	msg->refresh_timestamp();
-
-	routeable_message *rMsg = dynamic_cast<routeable_message *>(msg.get());
-	BroadcastMessage(*rMsg);
-
-	PLOG(logERROR) << " CARMACloud Plugin :: Broadcast tsm5:: " << tsm5ENC.get_payload_str();
+	//Get TCM id
+	Id64b_t tcmv01_req_id = tsm5message.get_j2735_data()->body.choice.tcmV01.reqid;
+	
+	ss.str(""); 
+    for(size_t i=0; i < tcmv01_req_id.size; i++)

```

Comment:
Can we fix this indentation

Reply:
![image](https://user-images.githubusercontent.com/62157949/158467229-94826a87-d5bc-47b5-b8c4-78b54619a928.png)